lizard_auth_server/tests/test_forms.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.test import TestCase
+from faker import Faker
+
+from lizard_auth_server.forms import JWTAuthenticationForm
+from lizard_auth_server.models import gen_key
+from lizard_auth_server.tests import factories
+
+fake = Faker()
+
+
+class JWTAuthenticationFormTestCase(TestCase):
+
+    def setUp(self):
+        self.portal = factories.PortalF()
+        self.user = factories.UserF()
+
+    def test_smoke_as_unknown_user(self):
```

Comment:
Je hebt hier een heleboel "smoke" tests, maar volgens mij zijn het gewoon normale tests, toch?

Een smoke test is "je stopt de stekker erin en kijkt of er rook uit komt". Dus even een view kaal aanroepen en kijken of dat in ieder geval niet crasht. Je gaat niet ook nog testen of de kleur van de rook goed is en of het wel lekker ruikt :-)